plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -519,6 +519,33 @@ public void testUnsupportedColumnType()
         session.execute("DROP KEYSPACE keyspace_6");
     }
 
+    @Test
+    public void testNullAndEmptyTimestamp()
+    {
+        String keyspace = "keyspace_7";
+        String tableName = "test_empty_timestamp";
+
+        session.execute("DROP KEYSPACE IF EXISTS " + keyspace);
+        session.execute(format("CREATE KEYSPACE %s WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor': 1}", keyspace));
+        session.execute(format("CREATE TABLE %s.%s (id int primary key, timestamp_column_with_null timestamp, timestamp_column_with_empty timestamp)", keyspace, tableName));
+        session.execute(format("INSERT INTO %s.%s (id, timestamp_column_with_null, timestamp_column_with_empty) VALUES (1, NULL, '')", keyspace, tableName));
+        assertContainsEventually(() -> execute("SHOW TABLES FROM cassandra." + keyspace), resultBuilder(getSession(), createUnboundedVarcharType())
```

Comment:
We don't do this in eg `testInsertToTableWithHiddenId`. why is it needed here?

Reply:
The difference is whether using both Trino and Cassandra native connection or not. All queries in `testInsertToTableWithHiddenId` happens in Trino, so we don't need to use `assertContainsEventually` method there.